### PR TITLE
Bump kubemacpool version to v0.5.0

### PR DESCRIPTION
**Pull request by alonSadan**
In the new version podDisruptionBudget and podAntiAffinity were added to kubemacpool.

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: data/kubemacpool/003-deployment.yaml [hunk 1]
```diff
@@ -57,3 +69,13 @@ spec:
               memory: 500Mi
       restartPolicy: Always
       terminationGracePeriodSeconds: 5
+---
```
Comment (SchSeba) [inline data/kubemacpool/003-deployment.yaml]: please move this to a new file

**Comment by SchSeba** [comment]
/hold 

we need to wait for a new release on the kubemacpool

**Comment by SchSeba** [comment]
ci test please

**Comment by phoracek** [comment]
ci test please

**Comment by phoracek** [comment]
@SchSeba @alonSadan 

```
12:33:32                message: 'could not apply (policy/v1beta1, Kind=PodDisruptionBudget) /mac-controller-manager:
12:33:32                  could not create (policy/v1beta1, Kind=PodDisruptionBudget) /mac-controller-manager:
12:33:32                  the server does not allow this method on the requested resource'
```

**Comment by SchSeba** [comment]
@phoracek yes we need k8s >= 1.13.

maybe we should add a check version?

**Comment by phoracek** [comment]
@SchSeba maybe, but I think we have `*` RBAC. My guess is, we have some issues in our `apply.go`.

**Comment by phoracek** [comment]
ci add to whitelist

**Comment by SchSeba** [comment]
ci test please

File: data/kubemacpool/003-deployment.yaml [hunk 2]
```diff
@@ -1,4 +1,3 @@
----
```
Comment (SchSeba) [inline data/kubemacpool/003-deployment.yaml]: please add this line again

**Comment by alonSadan** [comment]
ci test please

**Pull request merged**